build.gradle

```diff
@@ -9,11 +9,13 @@ plugins {
     id 'maven'
 }
 
-def buildNumber = System.getenv("TRAVIS_BRANCH") ?: "DEV-SNAPSHOT"
+def buildNumber = github.ref
+
+env:
```

Comment:
this is an env variable now, you don't need this here 👅

Reply:
Yeh, didn't mean to commit that